clap_builder/src/builder/arg.rs

```diff
@@ -69,8 +69,10 @@ pub struct Arg {
     pub(crate) r_unless_all: Vec<Id>,
     pub(crate) short: Option<char>,
     pub(crate) long: Option<Str>,
-    pub(crate) aliases: Vec<(Str, bool)>, // (name, visible)
-    pub(crate) short_aliases: Vec<(char, bool)>, // (name, visible)
+    pub(crate) aliases: Vec<(Str, bool)>,
+    // (name, visible)
+    pub(crate) short_aliases: Vec<(char, bool)>,
+    // (name, visible)
```

Comment:
Please revert as this is incorrect and unrelated

If anything, the comments should come before, not after